tests/fast-integration/eventing-test/eventing-test.js

```diff
@@ -123,6 +130,45 @@ describe('Eventing tests', function() {
     });
   }
 
+  function testStreamNotReCreated() {
```

Comment:
Can you add a simple line that explains how we check if the stream was recreated? Old vs current timestamp.